### DISCUSSION TITLE
Mod Recommendations for commit

### DISCUSSION THREAD

**Discussion by KendrickLamar2003** [QuestCraftPlusPlus/QuestCraft#463] [General]
So I'd like to know what mods I should put in the Recommended Mods section. What mods do YOU recommend? Please note they should fit the following criteria:
A. 1.18.2 and 1.19.3 with Fabric Support
B. Shouldn't change the core gameplay 
C. Focuses on **improving** Minecraft, not adding to it.
D. Should be light-weight
Since the next update will likely add Vulkan to QCXR and improve(?) performance, this should add a lot more compatibility for mods like Distant Horizons (YAYYYYYYYYYYYYY).

**Comment by Eyndjl** [comment]
I using the "Better Fps - Render Distance"
Quite an interesting mod. Slightly changes the rendering system of chunks. It does not conflict with other mods, based on my experience of using this mod.
Basically, the mod adds an adjustment of the vertical loading of chunks.

(Support for Vulkan? HURRAY!)

**Reply by KendrickLamar2003** [reply]
Thank you for the suggestion!

**Comment by Eyndjl** [comment]
I can't say whether it changes the gameplay much, but I would also recommend the mod "Swinging through grass". Allows you to hit mobs that are in the grass, without breaking the grass itself. This will help in battles with monsters, making battles less annoying.
